src/OrleansProviders/SQLServer/CreateTables.sql

```diff
@@ -1,113 +1,1440 @@
-﻿DROP TABLE [dbo].[OrleansMembershipTable]
-DROP TABLE [dbo].[OrleansMembershipVersionTable]
-DROP TABLE [dbo].[OrleansRemindersTable]
-DROP TABLE [dbo].[OrleansSiloMetricsTable]
-DROP TABLE [dbo].[OrleansClientMetricsTable]
-DROP TABLE [dbo].[OrleansStatisticsTable]
-
-CREATE TABLE [dbo].[OrleansMembershipTable]
+﻿/*
+Project Orleans Cloud Service SDK ver. 1.0
```

Comment:
I think the name of the script should be changed too. Perhaps something like ``CreateOrleansTables_SqlServer.sql``?